Gems/Atom/Feature/Common/Code/Source/FrameCaptureSystemComponent.cpp

```diff
@@ -389,94 +389,81 @@ namespace AZ
             return !AZ::RHI::IsNullRHI();
         }
 
-        FrameCaptureId FrameCaptureSystemComponent::CaptureScreenshotForWindow(const AZStd::string& filePath, AzFramework::NativeWindowHandle windowHandle)
+        FrameCaptureSystemComponent::CaptureHandle FrameCaptureSystemComponent::ScreenshotPreparation(
+            const AZStd::string& imagePath, AZ::RPI::AttachmentReadback::CallbackFunction callbackFunction)
         {
             if (!CanCapture())
             {
-                return InvalidFrameCaptureId;
+                AZ_Error("FrameCaptureSystemComponent", false, "Frame capture is not avaible.");
```

Comment:
I am not sure if we want to report an error here. It seems the CanCapture() checks if it's null renderer. And we may want this behavior and doesn't want to treat it as an error.

Reply:
+1, not an error to fail a capture here.

Reply:
I'll remove it. But is there a reason to capture a frame while it is running a null renderer?